pkg/generate/app/sourcelookup.go

```diff
@@ -73,19 +73,10 @@ func IsPossibleSourceRepository(s string) bool {
 
 // IsRemoteRepository checks whether the provided string is a remote repository or not
 func IsRemoteRepository(s string) bool {
-	if !s2igit.New(s2iutil.NewFileSystem()).ValidCloneSpecRemoteOnly(s) {
-		return false
-	}
-	url, err := url.Parse(s)
-	if err != nil {
-		return false
-	}
-	url.Fragment = ""
-	gitRepo := git.NewRepository()
-	if _, _, err := gitRepo.ListRemote(url.String()); err != nil {
-		return false
-	}
-	return true
+	// We don't try to connect to the remote registry because we may not yet have
+	// credentials.
+
+	return s2igit.New(s2iutil.NewFileSystem()).ValidCloneSpecRemoteOnly(s)
```

Comment:
i'm not clear on why this is changing.  @csrwng had made some changes to new-app to attempt to determine if credentials might be needed to access the repo, and if so, warn the user of that situation.  Is this changing that behavior?

I would expect us to have one of the following behaviors for private repos:
1) do the same logic as the bc secret injector:  try to find an appropriate secret and use it
2) warn the user their repo requires a secret so things may or may not work depending on the rest of their configuration